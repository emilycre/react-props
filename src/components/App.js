import React from 'react';
import Header from './Header';
import Color from './Color';

export default function App() {
  return (
    <>
      <Header />
      <Color name="Navy" hex="#000080" red={0} green={0} blue={128}/>
    </>
  );
}
