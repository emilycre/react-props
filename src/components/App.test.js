import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App Component', () => {
  it('Renders an app', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
