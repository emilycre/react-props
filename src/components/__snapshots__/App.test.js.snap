// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Renders an app 1`] = `<ColorPicker />`;
